### PR TITLE
Bengali translation for GitHUB cheatsheet

### PR DESCRIPTION
**Pull request by bolaram**
Fully updated **Bengali** translation of GitHub. please do additional thing for creating pdf.

### Next Steps
Need to **merge** & take step for make a _pdf_ for this translated cheat-sheet

### CODE REVIEW

**Comment by bolaram** [comment]
@beardofedu & @brianamarie will you please merge this ? I'd like to added on this organization so i can contribute on more community based work !

File: downloads/bn-BD/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
+---
+layout: cheat-sheet
+title: গিটহাব গিট চিট শীট
+byline: গিট হচ্ছে একটি ওপেন সোর্স ড্রিস্ট্রিবিউটেড ভার্শন কন্ট্রোল সিস্টেম যা আপনার ল্যাপটপ বা ডেস্কটপে গিটহাব সুবিধাদি প্রদান করে। এই চিত শীটে সাধারনভাবে ব্যবহৃত গিট কমান্ড লাইনের নির্দেশনা দেওয়া আছে । 
+leadingpath: ../../
```
Comment (brntbeer) [inline downloads/bn-BD/github-git-cheat-sheet.md]: @bolaram currently this line is what's causing the css not to load. if you change this to `../../../` then it should work!

**Comment by brntbeer** [comment]
@bolaram though I can't read Bengali, I appreciate the pull request! Do you know of someone else who can review it just to give us at least one additional person to review the wording?

File: downloads/bn-BD/github-git-cheat-sheet.md (hunk 1 above)
Comment (bolaram) [inline downloads/bn-BD/github-git-cheat-sheet.md]: Thanks for your comment. I updated . Is it Okay now ?
I'll ask a reviewer to review my edit!
Thanks. :

**Comment by ToufiqAmin** [comment]
I'm Toufiq Amin,generally contributes in localization on open sources projects . I've read carefully and reviewed GitHub Bengali cheat-sheet and its okay! Thanks @bolaram 👍 🎉

**Comment by crichID** [comment]
@bolaram thank you so much for contributing another translation of the git and github cheat sheet! It is deeply appreciated.

@ToufiqAmin thank you for reviewing this translation!

@brntbeer it looks like the change you requested has been applied! Would you mind checking it out and dismissing the review so we can merge this newest addition?

**Comment by crichID** [comment]
:heart: @brntbeer thank you! 

I am going to go ahead and :ship: this!

**Comment by bolaram** [comment]
@crichID @brntbeer Thanks all of you ! can i be added to GitHub organization . I'm ready to help to contribute more on educational tutorial.

**Comment by brntbeer** [comment]
@bolaram you're welcome!

However, we cannot add you to the GitHub organization, because this is meant for full time employees! Contributing to projects like this one by browsing to the public repositories would be a perfect way to get started on making educational material great! ✨ 🎉

**Pull request merged**